### PR TITLE
Feature/add icons

### PR DESCRIPTION
**Pull request by Julia-Alison**
Added in sprites
removed double ups on functions and variables.
Updated styles

screen shot 2015-12-17 at 6 02 45 pm
screen shot 2015-12-17 at 6 06 27 pm
screen shot 2015-12-17 at 6 06 31 pm
screen shot 2015-12-18 at 2 06 57 pm
screen shot 2016-01-05 at 9 02 41 am
screen shot 2016-01-05 at 9 02 46 am

### CODE REVIEW

**Comment by psyren89** [comment]
Looks alright :+1:

**Comment by lordmortis** [comment]
seems legit, are we waiting on @pzi to review or summach?

**Comment by Julia-Alison** [comment]
@lordmortis yas.

**Comment by pzi** [comment]
I don't like the new colour scheme.

Can you explain your choices? To me, it looks like either a job half done, not thought through and/or unfinished.

File: app/assets/stylesheets/admin/global/_forms.sass [hunk 1]
```diff
@@ -24,7 +24,7 @@ fieldset
 select[multiple=multiple],
 textarea
   margin-bottom: ($base-spacing / 2)
-  padding: ($base-spacing / 3)
+  padding: 6px 20px
```
Comment (pzi) [inline app/assets/stylesheets/admin/global/_forms.sass]: IMO 20px left and right padding is a lot. Looks like something is missing / has failed to load..

screen shot 2016-01-11 at 17 07 42

**Comment by jgunnink** [comment]
I actually really like the colour scheme :+1:, [PP] I guess? My only suggestion would be to darken the links which are brighter cyan to be the teal colour, same as the edit buttons. But yeah, I'm no designer :stuck_out_tongue:

**Comment by Julia-Alison** [comment]
I've updated the colour scheme the the standard flat ui colours. 
screen shot 2016-01-14 at 3 21 52 pm

**Pull request merged**